components/raftstore/src/coprocessor/split_check/mod.rs

```diff
@@ -128,3 +128,17 @@ impl<'a, E> Host<'a, E> {
         self.cfg.region_bucket_size.0
     }
 }
+
+#[inline]
+pub fn calc_split_keys_count(
```

Comment:
Any test case?

Reply:
the new added test_split_check_by_approximate_keys will call this method.

Reply:
I suggest to add a unit test case instead of depending on other cases.

Reply:
ok.

Reply:
> If the bucket version overflow, any method to detect such failure?

It's impossible to overflow practically,  even with 0.1s tick. Because to trigger refresh the buckets, it will need the enough write traffic to accumulate enough new data. 
But let's said the same term runs for 1000 years and it does overflow, then it will crash.  The restart will recover from it. 
On the other hand, if the region changes the leader every 0.1s tick (the region's term changed every 0.1s), then after  more than 13 years, the term << 32 will be back to 0. But practically it will be impossible.

Reply:
Why do you think it's impossible for a cluster to run for 13 years? 🤣